qa/rpc-tests/excessive.py

```diff
@@ -40,116 +254,160 @@ def run_test (self):
         self.nodes[2].setexcessiveblock(1000, 2)
         self.nodes[3].setexcessiveblock(1000, 3)
 
+        logging.info("Test excessively sized block, not propagating until accept depth is exceeded")
         addr = self.nodes[3].getnewaddress()
-        for i in range(0,20):
-          self.nodes[0].sendtoaddress(addr, 1.0)
+        self.repeatTx(20,self.nodes[0],addr)
+        counts = [ x.getblockcount() for x in self.nodes ]
+        base = counts[0]
+        logging.info("node0")
         self.nodes[0].generate(1)
         time.sleep(2) #give blocks a chance to fully propagate
```

Comment:
see my comment on fixed sleeps above. As this is old code (and the pattern repeats in the historic code in this test, consider perhaps a sync method which can help)